File: Competitive Coding - 1/2. Problems on Strings/Q3.py
import sys

s = sys.argv[1]
l = int(sys.argv[2])

def isEqualCharacters(s, l):
	# Write code from here
	rev=""
	for i in s:
		rev = i+rev
	first=s[:l]
	last=rev[l-1::-1]
	if first==last[::-1]:
		return True
	else:
		return False
	# ends here

print("true" if isEqualCharacters(s, l) else "false");

File: Competitive Coding - 1/1. Problems on Array/Q3.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
int middleElementOfArray(int arr[], int arrLen) {
    // write code from here
     return arrLen%2==0 ? arr[arrLen/2-1] : arr[arrLen/2];
     // ends here
}
int readIntArray(char *argsArray, int arr[]) {
    int col = 0;
    char *token = strtok(argsArray, ",");
    while (token != NULL) {
        arr[col] = atoi(token);
        token = strtok(NULL, ",");
        col++;
    }
    return col;
}
int main(int argc, char *argv[]) {
    int arr[strlen(argv[1])];
    int arrLen = readIntArray(argv[1], arr);
    printf("%d\n", middleElementOfArray(arr, arrLen));
    return 0;
}
